airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -428,16 +430,18 @@ def cleanup(self, pod: k8s.V1Pod, remote_pod: k8s.V1Pod):
                 with _suppress(Exception):
                     for event in self.pod_manager.read_pod_events(pod).items:
                         self.log.error("Pod Event: %s - %s", event.reason, event.message)
-            with _suppress(Exception):
-                self.process_pod_deletion(pod)
+            if remote_pod is not None:
```

Comment:
I'm not sure if we care, but if the create succeeds but the find fails, we can leave the pod with this approach.

Reply:
Is it fair to assume that that if the `pod find` fails then the pod doesn’t exist and we don’t need to delete it?